drake/systems/plants/constraint/EulerConstraint.m

```diff
@@ -7,11 +7,11 @@
     null_constraint_rows
     avg_rpy
   end
-  
+
   methods(Abstract,Access = protected)
     [rpy,J] = evalrpy(obj,kinsol);
   end
-  
+
   methods(Access = protected)
     function [c,dc] = evalValidTime(obj,kinsol)
```

Comment:
I would suggest to rename this function to `eval`, or merge it into the `eval` function